Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,18 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  return maartjesTasks.map(task => task.duration)
+  .map(durationInMinutes => durationInMinutes/60)
```

Comment:
this reads like a beautiful piece of code, doesn't it? :) great job!
Small remark: indent the method calls

Reply:
Ok noted. Thanks for the review. I haven't figured out why my eslint plugin on atom doesn't work properly.